scripts/design-system/agent-eval/intent-retrieval-eval.mjs

```
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+/**
+ * Golden intent retrieval eval — ranks @dt components for fixed queries.
+ * Uses the same rankComponentsForIntent() as find-component and MCP.
+ */
+import { readFileSync, existsSync } from "node:fs";
+import { join, resolve, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+import { rankComponentsForIntent } from "../usage-scan-lib.mjs";
+
+const ROOT = resolve(dirname(fileURLToPath(import.meta.url)), "../../..");
+const MANIFEST = join(ROOT, "nextjs-app/shared/foundations/dist/agent-manifest.json");
```

ESLint

[error] Replace `ROOT,·"nextjs-app/shared/foundations/dist/agent-manifest.json"` with `⏎··ROOT,⏎··"nextjs-app/shared/foundations/dist/agent-manifest.json",⏎`

(prettier/prettier)